study.md

```diff
@@ -38,29 +38,43 @@ As it makes sense to you please write the following in JSON format:
 - Has lived in two places recently: 123 Fake street, and 2 Muchinfo Road
 
 ```json
-<!-- your answer here -->
+"Jason":[
```

Comment:
ehhh, it would be pretty had to access a specific value if they all have the same key of "has" right?